test/game/game.js

```diff
@@ -110,37 +153,37 @@ describe("Game Server", function(){
       client6.disconnect();
       done();
     };
-    var expectStartGame = function() {
+    const expectStartGame = function () {
       client1.emit('startGame');
-      client1.on('gameUpdate', function(data) {
-        data.state.should.equal("waiting for players to pick");
+      client1.on('gameUpdate', (data) => {
+        data.state.should.equal('waiting for czar to draw a card');
       });
-      client2.on('gameUpdate', function(data) {
-        data.state.should.equal("waiting for players to pick");
+      client2.on('gameUpdate', (data) => {
+        data.state.should.equal('waiting for czar to draw a card');
       });
-      client3.on('gameUpdate', function(data) {
-        data.state.should.equal("waiting for players to pick");
+      client3.on('gameUpdate', (data) => {
+        data.state.should.equal('waiting for czar to draw a card');
       });
-      client4.on('gameUpdate', function(data) {
-        data.state.should.equal("waiting for players to pick");
+      client4.on('gameUpdate', (data) => {
+        data.state.should.equal('waiting for czar to draw a card');
       });
-      client5.on('gameUpdate', function(data) {
-        data.state.should.equal("waiting for players to pick");
+      client5.on('gameUpdate', (data) => {
+        data.state.should.equal('waiting for czar to draw a card');
       });
-      client6.on('gameUpdate', function(data) {
-        data.state.should.equal("waiting for players to pick");
+      client6.on('gameUpdate', (data) => {
+        data.state.should.equal('waiting for czar to draw a card');
       });
       setTimeout(disconnect, 200);
     };
-    client1.on('connect', function(data){
-      client1.emit('joinGame', { userID:'unauthenticated', room: '', createPrivate: true });
-      var connectOthers = true;
-      client1.on('gameUpdate', function(data) {
-        var gameID = data.gameID;
+    client1.on('connect', (data) => {
+      client1.emit('joinGame', { userID: 'unauthenticated', room: '', createPrivate: true });
+      let connectOthers = true;
+      client1.on('gameUpdate', (data) => {
+        let gameID = data.gameID;
```

Comment:
Can use `let` or `const` depending on whether it is re-assignable or not and have it as:
```js
const { gameID }  = data;
```

This will action the houndCI feedback